snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -153,7 +153,7 @@ open class Snapshot: NSObject {
 
         let networkLoadingIndicator = XCUIApplication().otherElements.deviceStatusBars.networkLoadingIndicators.element
         let networkLoadingIndicatorDisappeared = XCTNSPredicateExpectation(predicate: NSPredicate(format: "exists == false"), object: networkLoadingIndicator)
-        let _ = XCTWaiter.wait(for: [networkLoadingIndicatorDisappeared], timeout: timeout)
+        _ = XCTWaiter.wait(for: [networkLoadingIndicatorDisappeared], timeout: timeout)
```

Comment:
If it's discardable, do we even need the `_ = `?

Reply:
Having this declares that we tell the compiler that we know that something will be returned but we just don't care. Not having it causes one warning and using the `let _ =` pattern causes one as well. 

This way is the right way =)